modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtSiteLevel.testcase

```diff
@@ -0,0 +1,261 @@
+@component-name = "portal-batch-planner"
+definition {
+
+	property custom.properties = "feature.flag.COMMERCE-8087=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Import/Export";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given site scoped Student, Subject, and company scope University object definition with text field 'name' created") {
+			var studentId = ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "student",
+				en_US_plural_label = "students",
+				name = "Student",
+				requiredStringFieldName = "name",
+				scope = "site");
+
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "subject",
+				en_US_plural_label = "subjects",
+				name = "Subject",
+				requiredStringFieldName = "name",
+				scope = "site");
+
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "university",
+				en_US_plural_label = "universities",
+				name = "University",
+				requiredStringFieldName = "name");
+		}
+
+		task ("And Given many-to-many studentsSubjects relationship created") {
+			ObjectDefinitionAPI.createRelationshipWithObjectDefinitionNames(
+				childObjectName = "Subject",
+				deletionType = "cascade",
+				name = "studentsSubjects",
+				parentObjectName = "Student",
+				type = "manyToMany");
+		}
+
+		task ("And Given one-to-many studentAccounts relationship created") {
+			var accountId = ObjectDefinitionAPI.getObjectDefinitionIdByName(name = "AccountEntry");
+
+			ObjectDefinitionAPI.createRelationship(
+				deletionType = "cascade",
+				en_US_label = "StudentAccounts",
+				name = "studentAccounts",
+				objectDefinitionId1 = ${studentId},
+				objectDefinitionId2 = ${accountId},
+				type = "oneToMany");
+		}
+
+		task ("And Given object entries of Student, Subject, Account created with related entries") {
+			var accountEntryId = AccountAPI.createAccount(name = "accountForStudent");
+			var studentEntryId = ObjectDefinitionAPI.createObjectEntryWithName(
```

Comment:
please help check if we can create object entry and its related entry at the same time to reduce some code, thanks

Reply:
For this question, I have tested locally and it has some errors, I have asked it in the channel and I think we can keep it